docs/source/software/advanced_flight_controller.rst

```diff
@@ -1,3 +1,35 @@
 ==========================
 Advanced Flight Controller
 ==========================
+
+
+This page will go over how to use the trajectory tracking controller. The tracking controller is designed to follow a single trajectory from start to finish using a lower level velocity controller. This controller will run closed loop on the desired trajectory, but is **not** designed to accept a constantly changing trajectory. A trajectory can either be passed via a rostopic or as a ``.txt`` file
```

Comment:
Should there be sections on this page? Are there other controllers than the tracking controller?

Reply:
Before i wrote the documentation i was going to do the SE controller, but the pose and trajectory tracking controller are the only ones we'd tested in a "real" experiment so I figured it'd be unsafe to tell people the other controllers are usable. 

The though was that we'd expand the advanced controller section once we actually had more, e.g., the iLGR controller.